### PR TITLE
Changing the style of color swatches

### PR DESCRIPTION
**Pull request by j4n-co**

### CODE REVIEW

**Comment by j4n-co** [comment]
This pull request changes the style of the color swatches to the one pictured below
screen shot 2017-06-02 at 2 54 33 pm

File: css/wmui-style-guide.css [hunk 1]
```diff
@@ -961,27 +961,29 @@ code {
...
+	vertical-align: top;
 	margin-bottom: 3.2rem;
-	border: 1px solid #a2a9b1;
 	-moz-border-radius: 2px;
 	     border-radius: 2px;
 }
 .color--dark,
-	.color__wcag-level span {
+	.color__wcag-level span,
+	.color--dark .color-code {
```
Comment (Volker-E) [inline css/wmui-style-guide.css]: Why `.color--dark .color-code`, does `.color-dark` not apply?

File: css/wmui-style-guide.css [hunk 2]
```diff
@@ -961,27 +961,29 @@ code {
 	color: #000;
 	list-style: none none;
 	display: inline-block;
+	vertical-align: top;
```
Comment (Volker-E) [inline css/wmui-style-guide.css]: I'd like to point out https://www.mediawiki.org/wiki/Manual_talk:Coding_conventions/CSS/Archive_1#CSS.2FLess_property_order_proposal 
Let's please follow this…

File: css/wmui-style-guide.css [hunk 3]
```diff
@@ -961,27 +961,29 @@ code {
 	color: #000;
```
Comment (Volker-E) [inline css/wmui-style-guide.css]: We need to change the .dev.css version, that's what it's meant for, and output this file by Grunt build step.

File: css/wmui-style-guide.dev.css [hunk 4]
```diff
@@ -379,14 +380,15 @@ code {
 	}
 
 	.color__name {
-		font-weight: 700;
+		font-weight: 200;
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: Shouldn't this be 400? We've had the feedback that 200 is hard to read?

File: css/wmui-style-guide.css (hunk 3 above)
Comment (j4n-co) [inline css/wmui-style-guide.css]: Yeah, I didn't change the compiled file by hand. These changes are just the changes the build command outputted (should I not have run the build command?)

File: css/wmui-style-guide.css (hunk 1 above)
Comment (j4n-co) [inline css/wmui-style-guide.css]: on line [#399](https://github.com/wikimedia/WikimediaUI-Style-Guide/blob/master/css/wmui-style-guide.dev.css#L399) there is a `.color-code` styles that adds a color value, but I think it's better to just remove that then override it here.

File: css/wmui-style-guide.dev.css (hunk 4 above)
Comment (j4n-co) [inline css/wmui-style-guide.dev.css]: The revised style makes these color names huge ( `.color-name: font-size: 3rem;` ), so I don't  think readability will be an issue at this size.

File: css/wmui-style-guide.css (hunk 2 above)
Comment (j4n-co) [inline css/wmui-style-guide.css]: fixed

File: css/wmui-style-guide.css (hunk 1 above)
Comment (j4n-co) [inline css/wmui-style-guide.css]: fixed

**Comment by Ladsgroup** [comment]
I don't know if this PR is active or not but the picture you gave here is wrong, the light blue color is #eaf3ff (I know most of the color palette by heart) most of the gray ones are also wrong.

**Comment by Volker-E** [comment]
This has gotten a long way and was taken as base for 043edc9559024eb4ad56d2cf2da36de85c7aa823. Resolved #129 with commit and able to close this now 😄

**Pull request closed without merging**